architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -430,4 +631,196 @@ image:
+
+----
+oc describe is/python
+
+Name:			python
+Namespace:		default
+Created:		5 minutes ago
+Labels:			<none>
+Annotations:		openshift.io/image.dockerRepositoryCheck=2017-10-02T17:05:11Z
+Docker Pull Spec:	docker-registry.default.svc:5000/default/python
+Image Lookup:		local=false
+Unique Images:		1
+Tags:			2
+
+latest
+  tagged from python@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+
+  * centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+      About a minute ago
+
+3.5
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+      5 minutes ago
+----
+
+[[image-stream-mappings-working-adding-tags]]
+==== Adding Tags for an External Image
+
+Use the `oc tag` command for all tag-related operations, such as adding tags pointing to internal or external images:
+
+----
+oc tag <repositiory/image> <image-name:tag>
+----
+
+For example, this command maps the `docker.io/python:3.6.0` image to the `3.6` tag in the `python` image stream.
+
+----
+oc tag docker.io/python:3.6.0 python:3.6
+Tag python:3.6 set to docker.io/python:3.6.0.
+----
+
+If the external image is secured, you will need to create a secret with credentials for accessing that registry.
+See xref:../../dev_guide/managing_images.adoc#private-registries[Importing Images from Private Registries] for more details.
+
+[[image-stream-mappings-working-updating]]
+==== Updating an Image Stream Tag
+
+To update a tag to reflect another tag in an image stream:
+
+----
+oc tag <image-name:tag> <image-name:latest>
+----
+
+For example, the following updates the `latest` tag to reflect the `3.6` tag in an image stream:
+
+----
+oc tag python:3.6 python:latest
+Tag python:latest set to python@sha256:438208801c4806548460b27bd1fbcb7bb188273d13871ab43f.
+----
+
+==== Removing Image Stream Tags from an Image Stream
+
+To remove old tags from an image stream:
+
+----
+oc tag -d <image-name:tag>
+----
+
+For example:
+
+----
+oc tag -d python:3.5
+
+Deleted tag default/python:3.5.
+----
+
+
+[[image-stream-mappings-working-periodic]]
+==== Configuring Periodic Importing of Tags
+
+When working with an external Docker registry, to periodically re-import an image (such as, to get latest security updates), use the `--scheduled` flag:
+
+----
+oc tag <repositiory/image> <image-name:tag> --scheduled
+----
+
+For example:
+
+----
+oc tag docker.io/python:3.6.0 python:3.6 --scheduled
+
+Tag python:3.6 set to import docker.io/python:3.6.0 periodically.
+----
+
+This command causes {product-title} to periodically update this particular image stream tag. This period is a
+xref:../../install_config/master_node_configuration.adoc#master-config-image-config[cluster-wide setting] set to 15 minutes by default.
```

Comment:
@mburke5678 because this goes into the dedicated and online branches, it should have been conditionalized for enterprise and origin branches only. I have fixed it for dedicated and online.